src/hotspot/share/c1/c1_LinearScan.cpp

```diff
@@ -4599,6 +4605,10 @@ bool Interval::intersects_any_children_of(Interval* interval) const {
 
 #ifndef PRODUCT
 void Interval::print_on(outputStream* out) const {
```

Comment:
I suggest moving the impl of print_on to the declaration in the hpp-file. That will make it easier to see at a glance how the different print methods delegate to each other.

Reply:
That's a good idea - fixed it!